index.html

```diff
@@ -1,159 +1,284 @@
+              <li class="pro-lang">html</li>
+            </ul>
+            <button class="see-project" type="button">See Project</button>
+          </div>
+        </article>
+        <article class="project-holder">
+          <div class="image-holder">
+            <h3 class="multi-post">Multi-Post Stories <br />Gain+Glory</h3>
+            <ul class="language-section">
+              <li class="pro-lang">Ruby on rails</li>
+              <li class="pro-lang">css</li>
+              <li class="pro-lang">JavaScript</li>
+              <li class="pro-lang">html</li>
+            </ul>
+            <button class="see-project" type="button">See Project</button>
+          </div>
+        </article>
+      </section>
+      <section class="lang-frame-skills">
+        <article class="about-content">
+          <h2 class="about-me">About me</h2>
+          <p class="about-para">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <button class="resume-button" type="button">Get My Resume</button>
+        </article>
+        <article class="resume-hold">
+          <img src="./assets/skills/icon-languages.svg" alt="language-icon" />
+          <h3 class="resume-name">Languages</h3>
+          <ul class="skills-list">
+            <li class="green-button">JavaScript</li>
+            <li class="green-button">Ruby</li>
+            <li class="green-button">Html</li>
+            <li class="green-button">CSS</li>
+          </ul>
+        </article>
+        <article class="resume-hold">
+          <img src="./assets/skills/icon-frameworks.svg" alt="framework-icon" />
+          <h3 class="resume-name">Frameworks</h3>
+          <ul class="skills-list">
+            <li class="green-button">Bootstrap</li>
+            <li class="green-button">Ruby on rails</li>
+            <li class="green-button">RSpec</li>
+            <li class="green-button">Capybara</li>
+            <li class="green-button">Selenium</li>
+          </ul>
+        </article>
+        <article class="resume-hold">
+          <img src="./assets/skills/icon-skills.svg" alt="skills-icon" />
+          <h3 class="resume-name">Skills</h3>
+          <ul class="skills-list">
+            <li class="green-button">Codekit</li>
+            <li class="green-button">Github</li>
+            <li class="green-button">html</li>
+            <li class="green-button">Codepen</li>
+            <li class="green-button">Gitlab</li>
+            <li class="green-button">Terminal</li>
+          </ul>
+        </article>
+      </section>
+      <footer>
+        <p class="form-intro">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form class="contact-form">
```

Comment:
- [ ] Your form looks good but it is not working properly. To work the form needs the action and method attribute to integrate with [formspree service](https://formspree.io/html). Kindly read the getting started section and apply the formspree service to your form.

Reply:
Fixed.